lang/python/python3-readline/Makefile

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright (C) 2015 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
```

Comment:
You should omit it or assign it to yourself.
See: [CONTRIBUTING.md](https://github.com/openwrt/packages/blob/master/CONTRIBUTING.md#makefile-contents-should-contain)

Reply:
Argh, nuances.